README.md

```diff
@@ -223,3 +223,4 @@ Questions can be asked by raising an `Issue`.
 - [Anubhav Purohit](https://github.com/Anubhav-byte)
 - [Mitanshi Kshatriya](https://github.com/MitanshiKshatriya)
 - [Ganapathi Subramanyam Jayam](https://github.com/ganapathi12)
+- [**_Satyam Gadekar(@07Satyam)_**](https://github.com/07Satyam)
```

Comment:
Not alphabetics, but not a blocker, this can be improved in a future PR.

Please remove styling around your name, no one else has it bold and italic